website/content/v1.9/reference/api.md

```diff
@@ -127,6 +127,8 @@ description: Talos gRPC API reference.
 - [resource/definitions/hardware/hardware.proto](#resource/definitions/hardware/hardware.proto)
     - [MemoryModuleSpec](#talos.resource.definitions.hardware.MemoryModuleSpec)
     - [PCIDeviceSpec](#talos.resource.definitions.hardware.PCIDeviceSpec)
+    - [PCIDriverRebindConfigSpec](#talos.resource.definitions.hardware.PCIDriverRebindConfigSpec)
```

Comment:
let's please revert all changes in `website/` and re-build docs, they should land in v1.10